### ISSUE TITLE
Sending error messages to support

### ISSUE THREAD

**Issue by MichaelApproved** [KeithAndTheGirl/iOS#73]
Can you please add the ability for a listener to send an error message to support for assistance? For example, a listener received the following error but we can't figure out what is going wrong. 

https://www.dropbox.com/s/me9zlngy3gsubdd/cant%20find%20url%20on%20this%20server.jpg

It'd be great for there to be a button "Send error to support" with two buttons "OK" and "Skip"

OK would open an email with the following information prepopulated: 

TO: support@keithandthegirl.com
Subject: KATG iOS app v4 error
Body:
[Blank line to give listener an easy way to add more detail.]
The KATG app just had the following error:
Playback Failed
The requested URL was not found on this server

Helpful information about the error
Show name is such and such
Show number is whatever that would be
The URL trying to be downloaded
Other helpful information for us to debug here.

---

It would take a long time to get all the error messages in the app to use this new style of error reporting. For now, just add this to all episode playback error messages (both audio and video). Also add it to any errors people receive when streaming. If we get a lot of error reports in other sections, we'll add it elsewhere.

**Comment by MichaelApproved** [comment]
@Nick-R Have you been receiving error messages from crashes or errors?

**Comment by Nick-R** [comment]
No, i didn't. Did you send something? I remember we did some email reports but i don't remember how and can't find it in the code... Anyway i wrote about collecting crash reports in issue #67
